docs/_data/mesheryctlcommands/lifecycle/system/update/command/update.yaml

```diff
@@ -0,0 +1,4 @@
+name: update
+description: Pull new Meshery images from Docker Hub. Does not pulls new mesheryctl client. This command may be executed while Meshery is running.
```

Comment:
```suggestion
description: Pull new Meshery images from Docker Hub. Does not update `mesheryctl`. This command may be executed while Meshery is running.
```

Reply:
@Blakelist7 the above change is suggested :)

Reply:
Np